skrub/_dataframe/_common.py

```diff
@@ -299,7 +301,7 @@ def _all_null_like_pandas(col, length=None, dtype=None, name=None):
         dtype = col.dtype
     if name is None:
         name = col.name
-    return pd.Series(dtype=dtype, index=col.index, name=name)
+    return pd.Series(dtype=dtype, index=np.arange(length), name=name)
```

Comment:
I think we could keep the original index when length is not specified. not sure if we care

Reply:
You're right, we could address this in another PR if it comes up